### PR TITLE
180722100 - provide searcher on every page

### PR DESCRIPTION
**Pull request by kamtarnowski**
#### What's this PR do?
PR adds validator search button to navbar.

#### How should this be manually tested?
Verify the validator search button and the bar.

#### What are the relevant tickets?
[- URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/180722100)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?

### CODE REVIEW

File: app/javascript/packs/navigations/validator_searcher_btn_template.vue [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    methods: {
+      searcher_class() {
+        return this.is_active ? 'active' : ''
+      },
+      trigger_btn() {
+        this.is_active = !this.is_active
+
+        const validator_searcher_bar = document.getElementById("validator-searcher-bar")
+
+        if (this.is_active) {
```
Comment (psuchta) [inline app/javascript/packs/navigations/validator_searcher_btn_template.vue:26]: how about 
```suggestion
        validator_searcher_bar.classList.toggle("active")
```

Comment (psuchta) [inline app/javascript/packs/navigations/validator_searcher_btn_template.vue:26]: how about delete ifs and do only 
```suggestion
        validator_searcher_bar.classList.toggle("active")
```

**Review by psuchta — CHANGES_REQUESTED** [review]
Everything works fine except validator searching from the validator show page
/validators/validator_id
When I want to search for a validator from there, I am redirected to the home page

File: app/javascript/packs/navigations/validator_searcher_btn_template.vue (hunk 1 above)
Comment (kamtarnowski) [reply app/javascript/packs/navigations/validator_searcher_btn_template.vue:26]: It can work out. Thanks 👍

**Review by psuchta — APPROVED** [review]

**Review by kbiala — CHANGES_REQUESTED** [review]
Search toggle button is not working on mobiles.

**Comment by kamtarnowski** [comment]
@kbiala please double check it. I tried using Chrome with iphone resolutions (Devtools). All seems fine.

**Pull request merged**